rest-server/auth-server/src/main/resources/application.conf

```diff
@@ -1,11 +1,17 @@
+include "akka.conf"
+include "akka-http.conf"
+
+//uncomment line below to include ssl configuration
+//include "akka-ssl.conf"
```

Comment:
Really, uncomment and re-compile? We want to avoid recompiling.

Reply:
> Really, uncomment and re-compile? 
no, we use the external config file (it's -Dresource.config=... param). Dist script uses this file as a template. So, no recompilation is needed.

Reply:
But to change the setting `-Dresource.config=...` means re-compiling, correct? 

Ideally we want just a **restart** with new config to change SSL or Auth.

Reply:
> But to change the setting -Dresource.config=... means re-compiling, correct?

No, it's a file system path.

> Ideally we want just a restart with new config to change SSL or Auth.

Yeah, we can make some changes to the config (somewhere on the file system) and restart.